NuRadioReco/detector/test/test_rnog_detector.py

```diff
@@ -0,0 +1,28 @@
+from NuRadioReco.detector import detector
+import logging
+import datetime
+import os
+
+
+def test_detector():
+
+    det = detector.Detector(source="mongo", log_level=logging.DEBUG, always_query_entire_description=True,
+                            database_connection='RNOG_public', select_stations=24)
+    det.update(datetime.datetime(2023, 8, 2, 0, 0))
+    det.export("station24.json.xz")
+
+    det2 = detector.Detector(source="mongo", detector_file="station24.json.xz", select_stations=24)
+    det2.update(datetime.datetime(2023, 8, 2, 0, 0))
+
+    response = det.get_signal_chain_response(24, 0)
+
+
+    from NuRadioReco.framework import electric_field
+    with open(f"{os.path.dirname(__file__)}/test_ef.pkl", "rb") as f:
+        ef = electric_field.ElectricField(channel_ids=[0])
+        ef.deserialize(f.read())
```

Comment:
I'd probably just create a test-efield in the script (either some askaryan pulse or a delta pulse shouldn't require more than about 5 lines of code at most) rather than adding another pickle file, but I guess it's only 7 KB

Reply:
do you want to volunteer with an example?

Reply:
```
ef = electric_field.ElectricField(channel_ids=[0])
ef.set_frequency_spectrum(np.ones(1025, dtype=complex), sampling_rate=2.4)
```
if we don't care about polarization, otherwise shape (3,1025) and set the first row to 0.
If you'd prefer an Askaryan pulse let me know : ) I guess a  delta pulse (flat frequency spectrum) has the advantage of clearly showing the signal chain response in the frequency domain.